app/services/service_provider_updater.rb

```diff
@@ -7,6 +7,10 @@ class ServiceProviderUpdater
     updated_at
   ].to_set.freeze
 
+  SP_IGNORED_ATTRIBUTES = %i[
```

Comment:
**Question:** Should we also ignore this in `ServiceProviderSeeder`?

Reply:
No need IMO, because once we deploy the first PR, we can update the config repo to use plural, and there will be no more singular

Reply:
the reason for this is in case of a delay in deploying dashboard vs the IDP

Reply:
Ok that makes sense, thanks for clarifying!